.ci/setup_env_sles.sh

```diff
@@ -16,6 +16,12 @@ perl_repo="https://download.opensuse.org/repositories/devel:languages:perl/SLE_$
 sudo -E zypper addrepo --no-gpgcheck ${perl_repo}
 sudo -E zypper refresh
 
+echo "Add repo for myspell"
+leap_repo="http://download.opensuse.org/update/leap/15.0/oss/"
+leap_repo_name="leap-oss"
+sudo -E zypper ar -f --no-gpgcheck ${leap_repo} ${leap_repo_name}
```

Comment:
what about using `addrepo` instead of `ar`, just to maintain the consistency with above and below zypper commands.

Reply:
I also see that -f runs the `refresh` command, so not sure if the below line is needed? 
maybe @marcov can help here.

Reply:
-f makes the repo autorefresh, opposed to manually have to call the refresh command. I'd say, pick one or the other as you prefer :-)